priv/repo/migrations/20220701115243_create_peers_metrics.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Membrane.RTC.Engine.TimescaleDB.Repo.Migrations.CreatePeersMetrics do
+  use Ecto.Migration
+
+  @chunk_time_interval Application.get_env(:membrane_rtc_engine_timescaledb, Repo)[:chunk_time_interval] || "10 minutes"
+
+  def change do
+    create table(:peers_metrics, primary_key: false) do
```

Comment:
Why the `peer_id` isn't a primary key for this table?

Reply:
In this table each row contains metrics of single peer from single report, so when specific peer occurs in multiple reports, there will be many rows with `peer_id` equal to its id